lib/stdlib/src/erl_eval.erl

```diff
@@ -496,6 +523,26 @@ find_maxline(LC) ->
     _ = erl_parse:map_anno(F, LC),
     erase('$erl_eval_max_line').
 
+fun_used_bindings(Fun, Cs, Bs, FUVs) ->
+    {Used,InnerFUVs} =
+	case FUVs of
+	   %% If this clause is in our fun used vars tree,
```

Comment:
Nitpick: this comment is a bit misaligned, and there's a mix of tabs and spaces throughout the function.

Reply:
Fixed. I keep forgetting OTP is no longer doing emacs mode indentation!